apps/web/src/components/Shared/Markup/MarkupLink/index.tsx

```diff
@@ -19,7 +20,7 @@ const MarkupLink = ({ href, title = href }: MarkupLinkProps) => {
     return <Hashtag href={href} title={title} />;
   }
 
-  return <ExternalLink href={href} title={title} />;
+  return <ExternalLink href={href} title={title && truncateUrl(title, 30)} />;
```

Comment:
being title is not a boolean, i suggest to use ternary operator 🙇🏼

Reply:
Done!